pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -257,9 +257,9 @@ func (r *reconciler) add(rule *CompletedRule) error {
 
 	// Count up antrea_agent_ingress_networkpolicy_rule_count or antrea_agent_egress_networkpolicy_rule_count
 	if rule.Direction == v1beta1.DirectionIn {
-		metrics.IngressNetworkPolicyCount.Inc()
+		metrics.IngressNetworkPolicyRuleCount.Inc()
```

Comment:
I feel `IngressNetworkPolicyRuleCount` and `EgressNetworkPolicyRuleCount` could also be tracked in cache.go together with `NetworkPolicyCount ` as that's the source of reconciler.

Reply:
+1 Putting them together also makes it easily maintainable.

Reply:
Thank you for the comment.It sounds good. Let me see